src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -337,6 +337,20 @@ def node_exists(self, node_name):
         else:
             return True
 
+    def update_mon_count(self, newcount):
+        patch = [{"op": "replace", "path": "/spec/mon/count", "value": newcount}]
```

Comment:
We're just setting here a random variable in a `CephCluster` CRD. Would it make sense to restrict `newcount`? 
```python
if newcount not in [1,3,5,7]:
    raise ...
```
As we're specifying the target number here, I don't see a reason to support intermediate values, like `2`.

Reply:
Possibly. I wouldn't want to enumerate allowed values like that, but we could check that the value is not evenly divisible by 2?

Reply:
... and `>= 1`

Reply:
on the other hand, in a Rook controlled cluster, you shouldn't need that many MONs anyway. Even allowing more than 5 is a bit questionable.

Reply:
That sort of enforcement seems like it ought to be in rook itself, if it's necessary. I'm generally against putting up an artificial limit like that. Why wouldn't you want more than 5 mons in your 1000 node k8s cluster?